### ISSUE TITLE
Ritual of unbinding won't work after weak activation crystal was destroyed

### ISSUE THREAD

**Issue by Kacron** [WayofTime/BloodMagic#1241]
#### Issue Description:
This is part of feed the beast magic world 2.
In an unfortunate ritual accident I died and half of my items were destroyed including the activation crystal for my ritual of unbinding. Since then I've remade it, but it did not work. So I remade the ritual, and it didn't work. I just moved the ritual and it still does not work. This is a new activation crystal.

I've double checked the ritual layout. Any ideas what it might be? Am I SOL?
____
#### Affected Versions (Do *not* use "latest"):

- BloodMagic: unknown
- Minecraft: 1.6.4
- Forge: unknown

**Comment by WayofTime** [comment]
MC 1.6.4 is definitely no longer supported.

**Comment by Kacron** [comment]
Good evening, 
I've been playing blood magic through feed the beast. I made a forum post in the wrong section. Sorry about that, I don't know where else to go to. I haven't been able to do any new rituals since I lost the weak activation crystal for my ritual of unbinding. I've tried the ritual of suppression, and the ritual of full spring, with new activation crystals. Is there anything you know of that I can do to reverse this? Or is this a known issue?The feed the beast pack is magic world 2, minecraft version 1.6.4, I don't know the version of blood magic.ThanksKarina 

    On Saturday, March 3, 2018 12:27 PM, WayofTime <notifications@github.com> wrote:
 

 Closed #1241.—
